CMakeLists.txt

```diff
@@ -301,6 +301,7 @@ endforeach()
 # DOXYGEN. 
 #------------------------------------------------------------------------------
 Option(DOXYGEN_GENERATE_DOCUMENTATION ON)
+Option(DOXYGEN_OPTIONAL_INCLUDE_GENERATED ON)

```

Comment:
Please include a comment with the options that is then displayed as a tooltip in CMake GUI. See for example CMakeList of ExpressBindingGenerator, line 54.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/76f4b2a3bd0459949ff3212bc0bb85b6add91ff4